src/engine/sparqlExpressions/GroupConcatExpression.h

```diff
@@ -29,17 +31,29 @@ class GroupConcatExpression : public SparqlExpression {
     auto impl = [this, context](auto&& el)
         -> CPP_ret(ExpressionResult)(
             requires SingleExpressionResult<decltype(el)>) {
+      bool undefined = false;
```

Comment:
Can you (as part of the cleanup) move the implementation of the `GroupConcatExpression` into a `.cpp` file?